.travis.yml

```diff
@@ -1,20 +1,97 @@
-sudo: required
 matrix:
   include:
-    - os: linux
-      dist: xenial
+    - name: Quick Linux build
+      os: linux
+      dist: xenial  # Ubuntu 16.04, reasonably old and LTS
```

Comment:
Wouldn't this benefit from using a more modern LTS? The manylinux2014 is already an older platform.